<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformThing</class>
 <widget class="QWidget" name="PlatformThing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Platform Specific Configuration Module</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="shellChooser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string comment="'Windows' means the OS">Windows Desktop Shell</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="optNative">
        <property name="toolTip">
         <string extracomment="tooltip for option in switching Windows Desktop shells">Native Windows Explorer shell</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis for option in switching Windows Desktop shells">This is standard Windows Desktop shell. Choose this if you want to return your system to default desktop.</string>
        </property>
        <property name="text">
         <string extracomment="radio button to choose Windows Desktop shell">Native desktop shell</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="descrNative">
        <property name="text">
         <string>System's default desktop shell</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="optPlasma">
        <property name="toolTip">
         <string extracomment="tooltip for option in switching Windows Desktop shells">Use KDE desktop shell as it's seen on Linux</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis for option in switching Windows Desktop shells">Choose this option if you would like to use Plasma desktop shell for your Windows system.</string>
        </property>
        <property name="text">
         <string extracomment="radio button to choose Windows Desktop shell">Plasma desktop shell</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="descrPlasma">
        <property name="text">
         <string>KDE4 Plasma desktop shell</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="optCustom">
        <property name="toolTip">
         <string extracomment="tooltip for option in switching Windows Desktop shells">Your custom desktop shell</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis for option in switching Windows Desktop shells">Choose this and press &lt;i&gt;&quot;Setup...&quot;&lt;/i&gt; button to configure you custom desktop shell. Not recommended for average user.</string>
        </property>
        <property name="text">
         <string extracomment="radio button to chose Windows Desktop shell">Custom desktop shell</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="descrCustom">
        <property name="text">
         <string>This shell is reserved for the user's custom shell. Press the &quot;Setup&quot; button to setup your favorite shell.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnShellSetup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string extracomment="tooltip for button to setup custom Desktop shell">Press to setup your custom desktop shell</string>
          </property>
          <property name="whatsThis">
           <string extracomment="whatsThis for button to setup custom Desktop shells">Press this and configuration dialog will popup to let you configure your custom desktop shell</string>
          </property>
          <property name="text">
           <string>Setup...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="systemIntegration">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>System Integration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkDisableMenuGen">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">This will enable automatic regeneration of Windows' Start menu entries for KDE applications</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">This option defines if Windows' Start menu entries should be regenerated or not. By default start menu entries are regenerated automaticaly, but you may wish to disable it if you have issues with that.</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Enable automatic regeneration of start menu entries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUseNativeDialogs">
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">Use native Windows file dialogs instead of KDE ones</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">Choose this option to make KDE applications use usual Windows Open/Save file dialogs instead of custom KDE ones.</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Use native system file dialogs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkInstallCPl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">Install SystemSettings into Windows' Control Panel</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">This will install KDE SystemSettings as an element in Windows Control Panel.&lt;br&gt;&lt;b&gt;This option tweaks your Windows registry!&lt;/b&gt;</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Install as control panel element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkInstallCursors">
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">Install Oxygen cursors and as Windows cursor schemes</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">This will install Oxygen cursors into your system and combine them into cursor themes.&lt;br&gt;&lt;b&gt;This option tweaks your Windows registry!&lt;/b&gt;</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Install Oxygen cursor schemes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkInstallWallpapers">
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">This option installs KDE wallpapers into your &quot;My Pictures&quot; directory</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">This will install KDE wallpapers into &lt;i&gt;&quot;My Pictures&quot;&lt;/i&gt; directory so that you could use them as your Windows wallpapers. If the checkbox is set to half-checked then it means that there're new wallpapers available to update.&lt;br&gt; Wallpaper aspect ratio will be selected according to your current screen resolution.</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Install Oxygen wallpapers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkStartAtLogin">
        <property name="toolTip">
         <string extracomment="tooltip for checkbox">This will make essential KDE processes run at user login</string>
        </property>
        <property name="whatsThis">
         <string extracomment="whatsThis tooltip">Enabling this option will make needed KDE processes be started at user login. Normally these processes are started at when you start first KDE application during user session, so actual application launch is delayed.&lt;br&gt;It is recommended to enable this option to make your applications launch faster for the first time.&lt;br&gt;&lt;b&gt;This option tweaks your Windows registry!&lt;/b&gt;</string>
        </property>
        <property name="text">
         <string extracomment="checkbox caption in System Integration options">Load KDE at user login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>167</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
